main.cpp

```diff
@@ -110,6 +115,13 @@ void display_next_combo_name(size_t tries, const std::vector<std::string>& combo
 
 int main()
 {
+	Env finder(std::filesystem::current_path());
```

Comment:
This is **categorically not** the executable directory! It's the working directory. There is only one way to obtain the executable directory: through `argv[0]`.

Reply:
Would this be done via CMake?

Reply:
Not really, just modify main to receive the args:
```cpp
s32 main(s32 argc, char** argv)
```